tools/state-viewer/src/apply_chain_range.rs

```diff
@@ -364,7 +359,7 @@ pub fn apply_chain_range(
     let range = start_height..=end_height;
     let progress_reporter = ProgressReporter {
         cnt: AtomicU64::new(0),
-        ts: AtomicU64::new(timestamp()),
+        ts: AtomicU64::new(timestamp_ms()),
```

Comment:
let's also update the coment in ProgressReporter to say that this is in milliseconds. I'd also consider changing the name of the field (to timestamp_ms?)